tools/ci_build/github/apple/default_vision_os_framework_build_settings.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
current setup separate the xros/xrsimulator into a separate build_settings.json file. could potentially merge it into the full apple framework settings file to build a xcframework containing all supported platforms.

Reply:
for now let's keep it separate until we have the CI setup to be able to build visionOS. 

someone could manually merge xcframeworks from the full and visionOS builds if they wanted a combined one right? IIRC it's just a zip with a certain directory structure.

Reply:
> someone could manually merge xcframeworks from the full and visionOS builds if they wanted a combined one right? IIRC it's just a zip with a certain directory structure.

that's right. it can be manually assembled. ok, will just keep it as separated.